frontend/src/components/PPViewer/PPViewer.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import GoogleSlides from 'react-google-slides';
+
+interface IPPViewer {
+  link: string;
+}
```

Comment:
Do we really need?

Reply:
we're not exporting it even to container component, idk if it is a good idea

Reply:
needs to be discussed and agreed upon about types and folders, but I agree if it's a container-only local interface, it might be better to leave it inside the component

Reply:
done